Packages/vcs/Lib/VTKAnimate.py

```diff
@@ -122,8 +122,6 @@ def __init__(self,vcs_self):
         self.renderers = []
         self.last_size = None
         self.modified_listener = None
-        import atexit
-        atexit.register(self.close)
```

Comment:
@dlonie I'm about to test now. But did you check that pngs are now cleared? The reason this was put in is that the __del__ function was never reached hence leaving tons of temporary png file around.